File: solutions/ocaml/darts/2/darts.ml
let score (x: float) (y: float): int =
  let r : float =sqrt(x *. x +. y *. y) in
  match r with
    | r when r <= 1.0 -> 10
    | r when r <= 5. -> 5
    | r when r <= 10. -> 1
    | _ -> 0 

File: solutions/r/darts/2/darts.R
score <- function(x, y) {
  rsq = x^2 + y^2
  score<-if(rsq<=1){
    10 
  } else if(rsq<=25){
    5
  } else if (rsq<=100){
    1
  } else {
    0
  }
}
